packages/esm-tb-app/src/views/dashboard/patient-list-tabs/tb-patient-list-tabs.component.tsx

```diff
@@ -49,8 +50,14 @@ function TbHomePatientTabs() {
           {
             key: 'nextAppointmentDate',
             header: t('appointmentDate', 'Appointment Date'),
-            getValue: ({ latestEncounter }) => {
-              return getObsFromEncounter(latestEncounter, config.obsConcepts.nextAppointmentDate, true);
+            getValue:  (patient) => {
+              const patientLatestExtraEncounters = patient.latestExtraEncounters
+              if (patientLatestExtraEncounters && patientLatestExtraEncounters.length) {
+                const latestFollowUpEncounter = patientLatestExtraEncounters[0];
```

Comment:
i opt for using a filter or a find by the encounter type you want to use for this case follow up encounter. A requirement might come to add another `extraAssociatedEncounterTypes` and this might bring issues

Reply:
I had done it like that because in the list of `extraAssociatedEncounterTypes` we had only passed by one item but I'm going to implement @pirupius's solution. If I find by the `encounterType`, even if the `extraAssociatedEncounterTypes` array has more than one item and the order changes, the code won't break.

Reply:
Done